### PR TITLE
Improve contributing guide

### PR DESCRIPTION
**Pull request by nataliemarleny**
- Correct the path to the compiled Next binary
- Code to reveal which pages are avaliable within an example
- Preferred way of testing an external application with a locally compiled Next
- Add markdown code annotations

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -3,52 +3,90 @@
...
+## Running the integration apps
 
-```
-./node_modules/.bin/next ./test/integration/basic
-```
+The correct path to the compiled `next` binary can be discovered by running:
 
-## Testing in your own app
+```sh
+find . -name next -perm -u=x -type f
```
Comment (timneutkens) [inline contributing.md]: Did the `./node_modules/.bin/next` not exist? I can imagine lerna didn't work correctly and the file was not output 🤔

Comment (timneutkens) [reply contributing.md]: `./packages/next/dist/bin/next` is more correct though, so great change there 💯

Comment (nataliemarleny) [reply contributing.md]: Yes - the file did not exist 🤔

**Review by timneutkens — APPROVED** [review]

**Comment by timneutkens** [comment]
> Preferred way of testing an external application with a locally compiled Next

Awesome change 😌

File: contributing.md [hunk 2]
```diff
@@ -3,52 +3,90 @@
...
+```
 
-Because of the way Node.js resolves modules the easiest way to test your own application is copying it into the `test` directory.
+Running examples can be done with:
 
+```sh
+./packages/next/dist/bin/next ./test/integration/basic
+# OR
+./packages/next/dist/bin/next ./examples/basic-css/
```
Comment (ijjk) [inline contributing.md]:
```suggestion
yarn next ./examples/basic-css/
```

We could do this now that we added it in the package.json

**Pull request merged**